baseplate/experiments/__init__.py

```diff
@@ -90,6 +90,18 @@ def _get_experiment(self, name):
             self._experiment_cache[name] = experiment
         return self._experiment_cache[name]
 
+    def _get_all_experiment_names(self):
```

Comment:
It seems like this isn't used currently, can we leave it out until we actually need it?

Reply:
I can move it to another PR if necessary, but I'm working on something that depends on this now.

Reply:
That's fine, will there be a non-`_` method that uses this?  Or do we want to expose this directly?

Reply:
i'd suggest moving it to the other PR if that PR is what's going to need it. then it's able to change with the rest of that PR if needed.

Reply:
I think this should actually be public (not `_`), and will update accordingly. The "other PR" in this case is in a different repo (GraphQL).